docusaurus.config.js

```diff
@@ -255,9 +255,44 @@ module.exports = {
           { from: "/writing-rules/experiments/autofix/", to: "/writing-rules/autofix/" },
           { from: "/writing-rules/experiments/generic-pattern-matching/", to: "/writing-rules/generic-pattern-matching/" },
           { from: "/writing-rules/experiments/metavariable-analysis/", to: "/writing-rules/metavariable-analysis/" },
-          { from: "/deepsemgrep/", to: "/deepsemgrep/deepsemgrep-introduction/" },
-          { from: "/semgrep-app/integrations", to: "/semgrep-app/notifications/" },
-          { from: "/troubleshooting/gitlab-sast/", to: "/troubleshooting/semgrep-ci/" }
+
+          /* MAR 24 2023 POST LAUNCH REDIRECTS */
+
+          // Deepsemgrep
+          { from: "/deepsemgrep/"                         , to: "/semgrep-code/semgrep-pro-engine-intro/" }     ,
+          { from: "/deepsemgrep/deepsemgrep-introduction" , to: "/semgrep-code/semgrep-pro-engine-intro/" }     ,
+          { from: "/deepsemgrep/deepsemgrep-examples"     , to: "/semgrep-code/semgrep-pro-examples/" }  ,
```

Comment:
Change `/semgrep-code/semgrep-pro-examples/` to `/semgrep-code/semgrep-pro-engine-examples/`. (Spell the product name fully.)